### PR TITLE
configure add git clone example

### PR DESCRIPTION
**Pull request by Emyrk**
Also add "type" to the provider comment

### CODE REVIEW

File: workspaces/workspaces-as-code/templates.md [hunk 1]
```diff
@@ -64,13 +64,23 @@ workspace:
           command: |
             apt update
             apt install -y curl
+        # Be careful keyscans like this!
```
Comment (jawnsy) [inline workspaces/workspaces-as-code/templates.md:67]: is this comment misplaced? seems like it belongs above the "Add github to known hosts" step?

File: workspaces/workspaces-as-code/templates.md [hunk 2]
```diff
@@ -64,13 +64,23 @@ workspace:
...
+        - name: "Create organization directory"
+          command: "mkdir -p /home/coder/go/src/github.com/my-org"
+        - name: "Add github to known hosts"
```
Comment (jawnsy) [inline workspaces/workspaces-as-code/templates.md:70]:
```suggestion
        - name: "Add GitHub to known hosts"
```

**Review by jawnsy — APPROVED** [review]

File: workspaces/workspaces-as-code/templates.md (hunk 1 above)
Comment (Emyrk) [reply workspaces/workspaces-as-code/templates.md:67]: Whoops yes. That's what I get for copy pasting from my larger template example

**Review by khorne3 — APPROVED** [review]

**Review by IAmHughes — APPROVED** [review]
LGTM!

**Comment by Emyrk** [comment]
TIL the `h` in Git**H**ub is capitalized too

**Pull request merged**